ui/ui-frontend/projects/referential/src/app/file-format/file-format-preview/file-format-information-tab/file-format-information-tab.component.ts

```diff
@@ -96,38 +111,45 @@ export class FileFormatInformationTabComponent {
   }
 
   constructor(
+    @Inject(LOCALE_ID) private locale: string,
     private formBuilder: FormBuilder,
     private fileFormatService: FileFormatService,
     private route: ActivatedRoute,
     private securityService: SecurityService,
+    private startupService: StartupService,
```

Comment:
"private" pas nécessaire si utilisé uniquement dans le constructeur
```suggestion
    startupService: StartupService,
```